release/release.sh

```
@@ -7,6 +7,17 @@ dune subst
 
 dune build --profile release -p slipshow
 
+echo $(./_build/install/default/bin/slipshow --version)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)